dist/_locales/ko/messages.json

```diff
@@ -0,0 +1,232 @@
+{
+  "app_name": {
+    "message": "TabTabTab - Tab Manager",
+    "description": "The name of the application. Shown in the web store."
+  },
+  "app_name_short": {
+    "message": "TabTabTab",
+    "description": "The short name of the application. Shown at the top of the dropdown menu."
+  },
+  "app_desc": {
+    "message": "브라우저 탭 관리를 도와주는 확장 프로그램입니다.",
+    "description": "The short description of the application. Shown in the web store."
+  },
+  "window": {
+    "message": "Window",
+    "description": ""
+  },
+  "current_window": {
+    "message": "CurrentWindow",
+    "modified_message": "Opening Tabs",
+    "message": "열린 탭",
+    "description": ""
+  },
+  "pinned": {
+    "message": "고정됨",
+    "description": ""
+  },
+  "month": {
+    "message": "개월",
+    "description": ""
+  },
+  "months": {
+    "message": "개월",
+    "description": ""
+  },
+  "day": {
+    "message": "일",
+    "description": ""
+  },
+  "days": {
+    "message": "일",
+    "description": ""
+  },
+  "hour": {
+    "message": "시간",
+    "description": ""
+  },
+  "hours": {
+    "message": "시간",
+    "description": ""
+  },
+  "min": {
+    "message": "분",
+    "description": ""
+  },
+  "mins": {
+    "message": "분",
+    "description": ""
+  },
+  "sec": {
+    "message": "초",
+    "description": ""
+  },
+  "secs": {
+    "message": "초",
+    "description": ""
+  },
+  "ago": {
+    "message": "전",
+    "description": ""
+  },
+  "search_tabs": {
+    "message": "탭 검색",
+    "description": ""
+  },
+  "no_results_found": {
+    "message": "결과 없음",
+    "description": ""
+  },
```

Comment:
"no_result_found" is a text displayed when no results are found during a tab search. I think this translation is fine👍.

Reply:
When I tested, I got no message. Please check the screenshot I added. (`/dist/_locales/ko/no-message.png`)